README.md

```diff
@@ -18,3 +18,6 @@ the `#team-software` or `#team-analysis` channels).
 - [Hail Batch developer setup](hail_batch_dev.md)
 - [Setting up your terminal](terminal.md)
 - [Data sharing](data_sharing.md)
+- [Docker and Cloud Build](docker.md)
+- [IGV](bioinformatics/igv.md)
+- [Reference data](bioinformatics/reference_data.md)
```

Comment:
I think you accidentally added these two in this PR.

Reply:
Not an accident, just forgot to add links to the main README when submitting those two PRs. Hope it's okay to change with this one.

Reply:
Right, no worries.